habitat/sims/habitat_simulator.py

```diff
@@ -451,3 +454,18 @@ def distance_to_closest_obstacle(self, position, max_search_radius=2.0):
         return self._sim.pathfinder.distance_to_closest_obstacle(
             position, max_search_radius
         )
+
+    @property
+    def UP(self):
+        return np.array([0.0, 1.0, 0.0])
+
+    @property
+    def FORWARD(self):
+        return -np.array([0.0, 0.0, 1.0])
+
+    def straight_spath_points(self, position_a, position_b):
```

Comment:
Why we can't use geodesic distance based shortest path here, without creating new method?

Reply:
If you're asking why can't we do greedy search of best action using only geodesic distance/why do we need to call pathfinder to get path points the answer is that's slower, but also an option that the follower falls back on if it can't find this method.

Reply:
@danielgordon10 I thought this method `straight_spath_points ` returns points, not actions.

Reply:
It does return points. Then the shortest_path_finder uses those points to find actions.